### PR TITLE
NOISSUE - Add Change User Status Documentation

### PR DESCRIPTION
**Pull request by rodneyosodo**
### What does this do?
Add change user status to enabled and disabled for both `cli` and `api` documentations

### Which issue(s) does this PR fix/relate to?
None

### List any changes that modify/break current functionality
None

### Have you included tests for your changes?
No

### Did you document any new/modified functionality?
Yes

### Notes
N/A

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -25,24 +25,37 @@ Usage:
...
-  -m, --mainflux-url string    Mainflux host URL (default "http://localhost")
-  -o, --offset uint            offset query parameter
-  -t, --things-prefix string   Mainflux things service prefix
-  -u, --users-prefix string    Mainflux users service prefix
+  -l, --limit uint             Limit query parameter (default 100)
+  -m, --metadata string        Metadata query parameter
+  -n, --name string            Name query parameter
+  -o, --offset uint            Offset query parameter
+  -r, --raw                    Enables raw output mode for easier parsing of output
+  -S, --status string          Status query parameter
```
Comment (manuio) [inline docs/cli.md]: Should we clarify that it's only for users?

Comment (rodneyosodo) [reply docs/cli.md]: I don't think so. It is the same thing with `--email`. I am suggesting we change the description for cli i.e 

`-e, --email string Email query parameter` becomes `-e, --email string User email query parameter` and `-S, --status string Status query parameter` becomes `-S, --status string User status query parameter`

Comment (manuio) [reply docs/cli.md]: I agree

Comment (dborovcanin) [reply docs/cli.md]: OK, let's proceed this way. In this particular case, use lowercase `user email query parameter` and `user status query parameter`.

Comment (dborovcanin) [reply docs/cli.md]: @0x6f736f646f My bad, please capitalize. I thought `string User email query parameter` this is the message, not `User email query parameter`.

Comment (rodneyosodo) [reply docs/cli.md]: No problem

**Review by dborovcanin — DISMISSED** [review]

File: docs/cli.md [hunk 2]
```diff
@@ -172,7 +234,7 @@ mainflux-cli provision channels <file> <user_token>
 
 #### Update Channel
 ```bash
-mainflux-cli channels update '{"id":"<channel_id>","name":"myNewName"}' <user_token>
+mainflux-cli channels updatev '{"id":"<channel_id>","name":"myNewName"}' <user_token>
```
Comment (manuio) [inline docs/cli.md]: Fix typo `updatev`

File: docs/api.md [hunk 3]
```diff
@@ -166,6 +166,54 @@ Access-Control-Allow-Methods: *
...
+Date: Thu, 18 Aug 2022 13:28:20 GMT
+Content-Type: application/json
+Connection: keep-alive
+Strict-Transport-Security: max-age=63072000; includeSubdomains
+X-Frame-Options: DENY
+X-Content-Type-Options: nosniff
+Access-Control-Allow-Origin: *
+Access-Control-Allow-Methods: *
+Access-Control-Allow-Headers: *
+
```
Comment (manuio) [inline docs/api.md]: Remove this blank line

**Review by manuio — CHANGES_REQUESTED** [review]

File: docs/cli.md (hunk 2 above)
Comment (rodneyosodo) [reply docs/cli.md]: It is actually not a typo. But I will fix it

Comment (rodneyosodo) [reply docs/cli.md]: Fixed it in mainflux core repo by this PR https://github.com/mainflux/mainflux/pull/1649

File: docs/cli.md [hunk 4]
```diff
@@ -62,18 +75,42 @@ mainflux-cli channels -h
...
-  create      create <JSON_channel> <user_token>
-  delete      delete <channel_id> <user_token>
-  get         get <channel_id | all> <user_token>
-  update      update <JSON_string> <user_token>
+  connections   Connections list
+  create        Create channel
+  delete        Delete channel
+  get           Get channel
+  not-connected Not-connected list
+  update       Update channel
```
Comment (manuio) [inline docs/cli.md]: Fix the indentation

**Review by manuio — CHANGES_REQUESTED** [review]

File: docs/cli.md [hunk 5]
```diff
@@ -62,18 +75,42 @@ mainflux-cli channels -h
...
+  -l, --limit uint             Limit query parameter (default 100)
+  -m, --metadata string        Metadata query parameter
+  -n, --name string            Name query parameter
+  -o, --offset uint            Offset query parameter
+  -r, --raw                    Enables raw output mode for easier parsing of output
+  -S, --status string          User status query parameter
+  -t, --things-url string      Things service URL (default "http://localhost")
+  -u, --users-url string       Users service URL (default "http://localhost")
+
+Use "mainflux-cli channels [command] --help" for more information about a command.
```
Comment (manuio) [inline docs/cli.md]: remove this blank line

**Review by manuio — CHANGES_REQUESTED** [review]

File: docs/cli.md [hunk 6]
```diff
@@ -102,9 +138,14 @@ else
...
 ```
 
-#### Retrieve User
+#### Retrieve User By ID
 ```bash
-mainflux-cli users get <user_token>
+mainflux-cli users get <user_id> <user_token>
+```
+
+#### Retrieve All User
```
Comment (manuio) [inline docs/cli.md]: Fix `Retrieve All Users` and probably better `Retrieve Users` or `Get Users`. BTW, this is not consistent with `Get Channels` and `Get Things`

Comment (rodneyosodo) [reply docs/cli.md]: I will resort to using Get `xxx` as it has been implemented for `api` at https://github.com/mainflux/docs/pull/125

**Review by drasko — APPROVED** [review]
LGTM

**Review by dborovcanin — APPROVED** [review]

**Pull request merged**